requirements/build-dev-requirements.in

```diff
@@ -0,0 +1,3 @@
+mypy == 0.670
```

Comment:
Instead of introducing an artificial top-level `requirements` directory, let's keep requirement definition files per 'project'?

Reply:
Do you mean moving those files in the `buildchain` directory (would work, a bit dirty as `buildchain is a Python module and that's no place to put such files (and adding a extra directory level will make weird import), but would work).

Or did I misunderstand your suggestion?

Reply:
Done (move the requirements.txt in the `buildchain` directory)